tensorboard/webapp/plugins/npmi/views/annotations_list/annotation/annotation_component.ng.html

```diff
@@ -0,0 +1,48 @@
+<!--
+@license
+Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<div
+  class="annotation-container"
```

Comment:
remove this, if you can just style `:host`.

Reply:
`[ngClass]="{'selected-row': selectedAnnotations.includes(annotation)}"`
I think I cant do this then properly, right?